rust-toolchain

```diff
@@ -0,0 +1 @@
+stable
```

Comment:
maybe not good to keep it

Reply:
why?

Reply:
because it forces using stable to build in CI